lib/ViewModels/DataCatalogTabViewModel.js

```diff
@@ -47,7 +47,7 @@ DataCatalogTabViewModel.prototype.showInfo = function(item) {
 DataCatalogTabViewModel.prototype.clickEnabled = function(item) {
     item.terria.analytics.logEvent('dataSource', 'clickItem', item.name);
 
-    if (defined(item.initialMessage) && !item.isEnabled && !item.initialMessageViewed) {
```

Comment:
Doesn't this `!item.initialMessageViewed` bit need to be moved somewhere else?  Or if it's really not needed anymore, the property should be removed from `CatalogMember`.